docs/api/actions.md

```diff
@@ -104,6 +104,45 @@ function readPlanets () {
 repo.push(readPlanets)
 ```
 
+### Return a generator
+
+Often times we need to dispatch multiple actions in sequential
+order. For example, what if we want to ask the user to confirm their
+action before deleting a record?
+
+This can be accomplished by using a generator:
+
+```javascript
+function ask (message) {
+  return action => {
+    if (confirm(message)) {
+      action.resolve()
+    } else {
+      action.reject()
+    }
+  }
+}
+
+function deleteUser (id) {
+  return fetch.delete('/users/${id}').then(response => response.json())
+}
+
+function confirmAndDelete (user) {
+  return function * (send) {
```

Comment:
is there already a section in the docs about "dependencies" like this? might want to add a note about support for these here